### ISSUE TITLE
 Error commiting changes from processor: error in *processors.TransactionProcessor.Commit

### ISSUE THREAD

**Issue by abuiles** [stellar/go#2448]
We got the following error while ingesting some ledgers in verify-range 

> time="2020-04-03T20:43:27.901Z" level=fatal msg="Error running processors on ledger: Error commiting changes from processor: error in *processors.TransactionProcessor.Commit: Error flushing transaction batch: error adding values while inserting to history_transactions: exec failed: pq: value \"2650000000\" is out of range for type integer" pid=1091

> time="2020-04-03T19:33:27.099Z" level=fatal msg="Error running processors on ledger: Error commiting changes from processor: error in *processors.TransactionProcessor.Commit: Error flushing transaction batch: error adding values while inserting to history_transactions: exec failed: pq: value \"2500000000\" is out of range for type integer" pid=1097

## Ranges
FROM: 24737727 TO: 24744127
FROM: 24334527 TO: 24340927

**Comment by tamirms** [comment]
the errors occurred in ledgers 24338403 and 24740516
```
time="2020-04-03T19:33:26.920Z" level=info msg="Processing ledger" commit=true graph=true ledger=true pid=1097 sequence=24338403 service=expingest state=true
time="2020-04-03T19:33:27.099Z" level=fatal msg="Error running processors on ledger: Error commiting changes from processor: error in *processors.TransactionProcessor.Commit: Error flushing transaction batch: error adding values while inserting to history_transactions: exec failed: pq: value "2500000000" is out of range for type integer" pid=1097
```
```
time="2020-04-03T20:43:27.776Z" level=info msg="Processing ledger" commit=true graph=true ledger=true pid=1091 sequence=24740516 service=expingest state=true
time="2020-04-03T20:43:27.888Z" level=error msg="Error in ingestion state machine" current_state="verifyRange(fromLedger=24737727, toLedger=24744127, verifyState=true)" error="Error running processors on ledger: Error commiting changes from processor: error in *processors.TransactionProcessor.Commit: Error flushing transaction batch: error adding values while inserting to history_transactions: exec failed: pq: value \"2650000000\" is out of range for type integer" next_state=stop pid=1091 service=expingest
```

**Comment by abuiles** [comment]
Fixed in #2456
